samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1-object-interop.conf

```diff
@@ -0,0 +1,33 @@
+#Disable Aplication sensor Object and GNSS modules away
+CONFIG_APP_ACCELEROMETER=n
+CONFIG_APP_PUSH_BUTTON=n
+CONFIG_APP_ENV_SENSOR=n
+CONFIG_APP_TEMP_SENSOR=n
+CONFIG_APP_PRESS_SENSOR=n
+CONFIG_APP_HUMID_SENSOR=n
+CONFIG_APP_GAS_RES_SENSOR=n
+CONFIG_APP_LIGHT_CONTROL=n
+CONFIG_APP_LIGHT_SENSOR=n
+CONFIG_APP_ONOFF_SWITCH=n
+CONFIG_APP_GNSS=n
+
+#Enable Portfolio object
+CONFIG_LWM2M_PORTFOLIO_OBJ_SUPPORT=y
+
+#LwM2M v1.1 configure
+CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1280
+CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
+
+#Enable SenML Json content format
+CONFIG_BASE64=y
+CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
+
+#Enable extra conten format support
+CONFIG_LWM2M_RW_JSON_SUPPORT=y
+
+#Enable Dynamically Send Operation
+CONFIG_APP_LWM2M_CONFORMANCE_TESTING=y
+
+#Modem lib trace's control
+CONFIG_NRF_MODEM_LIB_TRACE_ENABLED=n
```

Comment:
Why do we need two overlays? These seem largely the same.

Reply:
Core cover int-1-311 and yes only diffrence is that Core disable Signal measurement and Connection monitoring object. That it can fit all for / root read. Conformance test test define 9 different setup. So 2 is much smaller number.